build/distributions/CentOS/6/x86_64/DVD/ks/dvd/include/simp_bootstrap_config

```diff
@@ -1,44 +0,0 @@
-security_relevant_logs="if \$syslogfacility-text == 'authpriv' or \$syslogfacility-text == 'local5' or \$syslogfacility-text == 'local6' or \$syslogfacility-text == 'local7' or \$syslogpriority-text == 'emerg' or ( \$syslogfacility-text == 'kern' and \$msg startswith 'IPT:' ) then"
```

Comment:
Are there any ramifications to removing this file?  I can see that the settings that seemed related to simp config are wildly out of date, and thus not apparently used.  However, it is unclear about other config such as security_relevant_logs.